lib/broadway_kafka/producer.ex

```diff
@@ -677,4 +710,21 @@ defmodule BroadwayKafka.Producer do
   defp schedule_reconnect(timeout) do
     Process.send_after(self(), :reconnect, timeout)
   end
+
+  defp log_potential_rebalancing_warning(state) do
+    if state.draining_after_revoke_started_at do
+      finished_at = System.monotonic_time(:second)
+
+      rebalancing_timeout =
+        Keyword.get(state.config[:group_config], :rebalance_timeout_seconds, 30)
+
+      diff = finished_at - state.draining_after_revoke_started_at
+
+      if(diff > rebalancing_timeout) do
+        Logger.warn(
+          "#{state.client_id}: It took #{diff} seconds to commit the offsets while waiting for acks. You might need to adjust :rebalance_timeout_seconds option for you case."
```

Comment:
Instead of mentioning `rebalance_timeout_seconds` I can give a link to the doc `Consumer Rebalancing` section.

Reply:
WDYT about something like:

Committing offsets took #{diff} seconds, which is more than :rebalancing_timeout_seconds of #{timeout}s. This may cause XYZ. Please read the "Consumer Rebalancing" section in BroadwayKafka.Producer docs for more information"